internal/pkg/proofs/election_poster.go

```diff
@@ -51,3 +54,13 @@ func (ep *ElectionPoster) GenerateEPostCandidates(sectorInfo sector.SortedSector
 	}
 	return candidates, nil
 }
+
+// ElectionPostChallengeCount is the total number of partial tickets allowed by
+// the system
+func (ep *ElectionPoster) ElectionPostChallengeCount(sectors, faults uint64) uint64 {
+	if sectors-faults == 0 {
+		return 0
+	}
+	// ceil(sectors / SectorChallengeRatioDiv)
+	return (sectors-faults-1)/SectorChallengeRatioDiv + 1
```

Comment:
Does this pass gofmt? Can we place some spaces between the `-` and also the `/`.